ext/sockets/tests/bug46360.phpt

```diff
@@ -1,17 +1,13 @@
 --TEST--
 Bug 46360 - TCP_NODELAY constant (sock_get_option, sock_set_option)
+--SKIPIF--
+<?php if (!extension_loaded('sockets')) die('skip sockets extension not loaded'); ?>
 --CREDITS--
 Florian Anderiasch
 fa@php.net
---SKIPIF--
-<?php
-    if (version_compare(phpversion(), '5.2.7', '<')) {
-        die('skip old php, not eligible');
-    }
-?>
 --FILE--
 <?php
-    var_dump('TCP_NODELAY');
+    var_dump(TCP_NODELAY);
 ?>
 --EXPECT--
-string(11) "TCP_NODELAY"
+int(1)
```

Comment:
The value of TCP_NODELAY is probably doing to be system-dependent, so checking for `int(%d)` here would be better.